test/browser/lifecycles/shouldComponentUpdate.test.js

```diff
@@ -764,4 +764,65 @@ describe('Lifecycle methods', () => {
 		rerender();
 		expect(scratch.innerHTML).to.equal('<p>bar</p>');
 	});
+
+	it('should reorder non-updating nested Fragment children', () => {
```

Comment:
Will fail until we find a performant solution to https://github.com/preactjs/preact/pull/2551/files#r439372897